### DISCUSSION TITLE
Dynamic view and animation

### DISCUSSION THREAD

**Discussion by KpTn6974** [structurizr/java#361] [Q&A] [answered]
Hello,

In the [animation documentation page](https://docs.structurizr.com/ui/diagrams/animation) it is said that animation is relationship based for dynamic views.

But in the [animation language reference](https://docs.structurizr.com/dsl/language#dynamic-view) animation is not listed as a valid property for dynamic view.

And when I tried something like that, I have the following error "The relationship "animation" does not exist at line 65: animation"

```
views {
    dynamic mySystem {
        title "My dynamic diagram"
        step1 = user -> containerA "My description"
        step2 = containerA -> containerB "My other description"
        animation {
            step1
        }
        animation {
            step2
        }
    }
}
```

Am I missing something ? Is it really possible to combine dynamic view and animation ?

**Comment by simonbrowndotje** [comment, marked as answer]
Animation is enabled by default - no need to add a separate `animation` block.

**Comment by KpTn6974** [comment]
Thanks for the answer.

I should have tried the keyboard shortcuts !
